src/helpers/buildWalletSections.js

```diff
@@ -439,9 +429,9 @@ const sortImagesToPreload = images => {
   const filtered = compact(flattenDeep(images));
   const grouped = groupBy(filtered, property('priority'));
   return [
-    ...get(grouped, 'high', []),
-    ...get(grouped, 'normal', []),
-    ...get(grouped, 'low', []),
+    ...(grouped?.high ?? []),
+    ...(grouped?.normal ?? []),
+    ...(grouped?.low ?? []),
```

Comment:
```suggestion
    ...grouped?.high,
    ...grouped?.normal,
    ...grouped?.low,
```

I think this is the same

Reply:
yea, just Prettier argued about it

Reply:
Since it passes the CI i would leave it as is